src/components/layout/Footer.tsx

```
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+import { Facebook, Twitter, Linkedin, Mail, Phone, MapPin } from 'lucide-react';
+
+const Footer = () => {
+    return (
+        <footer>
+            <div className="bg-theme-night text-theme-gray-900 flex flex-col gap-[50px] px-[162px] py-[50px] relative">
+                <div className="flex items-center justify-between relative">
+                    {/* Left: Logo */}
+                    <h4 className="flex-shrink-0">ResuMATE</h4>
+                    {/* Center: Nav */}
+                    <div className='flex gap-[30px] absolute left-1/2 -translate-x-1/2 top-1/2 -translate-y-1/2'>
```

ESLint

[error] Replace `····················<div·className='flex·gap-[30px]·absolute·left-1/2·-translate-x-1/2·top-1/2·-translate-y-1/2'` with `··········<div·className="absolute·left-1/2·top-1/2·flex·-translate-x-1/2·-translate-y-1/2·gap-[30px]"`

(prettier/prettier)